internal/controller/predicates/predicates_test.go

```diff
@@ -0,0 +1,210 @@
+package predicates
```

Comment:
I am confused, what have those test to do with the changes in this PR ?

Reply:
Nothing, I forgot to push it in an earlier PR

Reply:
Ok happens, fine